code/game/objects/items/explosives/bombvest.dm

```diff
@@ -18,6 +35,12 @@
 	if(activator.wear_suit != src)
 		to_chat(activator, "Due to the rigging of this device, it can only be detonated while worn.") //If you are going to use this, you have to accept death. No armor allowed.
 		return FALSE
+	if(istype(activator.l_hand, /obj/item/weapon/shield/riot) || istype(activator.r_hand, /obj/item/weapon/shield/riot) || istype(activator.back, /obj/item/weapon/shield/riot))
```

Comment:
why does it check for back though back doesnt protect

Reply:
if you just face north you look like a regular unga?

Reply:
And there are plenty of maps where pushing only north is valid until you run into a xeno that tries to melee you.